packages/dx-scheduler-core/src/types/drag-drop.types.ts

```diff
@@ -28,7 +28,7 @@ type AppointmentBoundaries = {
 };
 
 export type CalculateAppointmentTimeBoundaries = PureComputed<
-  [AppointmentModel, AppointmentModel, string, number, number, number],
+  [AppointmentModel, ViewCell | AllDayCell, string, number, number, number],
```

Comment:
Almost all the types in scheduler-core/types should be private - please add @internal modifier to exports in types/index